src/ui/hooks/usePowerState.ts

```diff
@@ -137,11 +137,15 @@ class BatteryLevelSubject {
 
   start() {
     this.intervalHandler = setInterval(() => {
-      ExpoBattery.getBatteryLevelAsync().then(batteryLevel => {
-        Object.values(this.listeners).forEach(listener => {
-          listener(batteryLevel);
+      ExpoBattery.getBatteryLevelAsync()
```

Comment:
just squeezing this in.. the current error that I'm looking at is a promise rejection here, so have added a log entry.